spec/zoneddatetime.html

```diff
@@ -1262,14 +1309,24 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-differencezoneddatetime" aoid="DifferenceZonedDateTime">
-      <h1>DifferenceZonedDateTime ( _ns1_, _ns2_, _timeZone_, _calendar_, _largestUnit_ [ , _options_ ] )</h1>
-      <p>
-        The abstract operation DifferenceZonedDateTime computes the difference between two exact times expressed in nanoseconds since the Unix epoch, and balances the result so that there is no non-zero unit larger than _largestUnit_ in the result, taking calendar reckoning and time zone offset changes into account.
-      </p>
+    <emu-clause id="sec-temporal-differencezoneddatetime" type="abstract operation">
+      <h1>
+        DifferenceZonedDateTime (
+          _ns1_: a BigInt value,
+          _ns2_: a BigInt value,
```

Comment:
do we say `a BigInt value` here, as opposed to just `a BigInt`?

Reply:
There's a couple already in use, but I don't mind changing this:
https://github.com/tc39/proposal-temporal/blob/f47ebd8475fa016d60e6e8f44a03b5ea4f14de5f/spec/zoneddatetime.html#L1225
https://github.com/tc39/proposal-temporal/blob/f47ebd8475fa016d60e6e8f44a03b5ea4f14de5f/spec/instant.html#L570

Reply:
as long as ecmarkup supports it, the more concise form seems better :-)

Reply:
Done :+1: